diagnostic.md

```diff
@@ -176,6 +205,6 @@ with the class `.stretch` be at a _screen width_ of 550px?
 }
 ```
 
-<!-- your answer starts here -->
+it would be 240px
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have ```.stretch``` be 25% and at 550px width, 25% is 137.5 px.